.github/workflows/buildAndPushImage.yml

```diff
@@ -23,11 +23,29 @@ jobs:
         with:
           submodules: 'true'
 
-      - name: Login to GitHub Container Registry
-        run: echo ${{ secrets.GITHUB_TOKEN }} | docker login ghcr.io -u well5a --password-stdin
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v3
+        with:
+          platforms: linux/amd64,linux/arm64
 
-      - name: Build the Docker image
-        run: docker build . --file Dockerfile --tag $IMAGE_URL/$IMAGE_NAME:$IMAGE_TAG
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v3
+        with:
+          platforms: linux/amd64,linux/arm64
```

Comment:
shouldn't we remove the arm64 tag here and in line 48?

Reply:
Was resolved with latest commit 👍🏼